frontend/src/App.tsx

```diff
@@ -2,15 +2,16 @@ import React from 'react';
 import './App.css';
 import { ChakraProvider } from '@chakra-ui/react';
 import { Provider } from 'react-redux';
-import OntologyTable from './components/OntologyTable';
 import ErrorModal from './components/ErrorModal';
 import store from './state/store';
+import Graph from './components/Graph';
 
 const App = () => (
   <Provider store={store}>
     <ChakraProvider>
       <div className="App">
-        <OntologyTable />
+        <Graph />
+        {/* <OntologyTable /> */}
```

Comment:
Even though we might use OntologyTable at a later stage, this comment should be removed.